helm/charts/infra/templates/connector/deployment.yaml

```diff
@@ -31,7 +31,14 @@ spec:
 {{- toYaml .Values.connector.securityContext | nindent 12 }}
           image: "{{ include "connector.image.repository" . }}:{{ include "connector.image.tag" . }}"
           imagePullPolicy: {{ include "connector.image.pullPolicy" . }}
-          args: ["connector", "-f", "/etc/infrahq/infra.yaml"]
+          args:
```

Comment:
This seems kind of confusing to someone opening the helm chart without the context that we want to be able to set the log level from an external config file. Maybe a comment would help that?